File: SecondYear/anshika1812_AnshikaChhabra_2226CSE1186_2/Week_3/Linear Search.c
#include<stdio.h>

int main(){
    int arr[1000],n,i,num;
    scanf("%d %d\n",&n,&num);
    for(i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    for(i=0;i<n;i++){
        if(arr[i]==num){
            return i;
        }
    }
    return -1;    
}

File: SecondYear/praveer7398_PraveerNandan_2226ce1011_2/Week_3/linear_search.java
import java.util.Scanner;

public class linear_search {
    public static int findIndex(int[] arr, int num) {
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == num) {
                return i; 
            }
        }
        return -1; 
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        
        int n = scanner.nextInt();
        int[] arr = new int[n];
        
        int numToFind = scanner.nextInt();

        
        for (int i = 0; i < n; i++) {
            arr[i] = scanner.nextInt();
        }

       

        int result = findIndex(arr, numToFind);

        if (result != -1) {
            System.out.println(result);
        } else {
            System.out.println("-1");
        }

        scanner.close();
    }
}
